src/Components/Facility/ConsultationDetails.tsx

```diff
@@ -402,7 +402,11 @@ export const ConsultationDetails = (props: any) => {
             </div>
 
             <div id="discharge-notes-div">
-              <InputLabel id="refered-label">Discharge Notes</InputLabel>
+              <InputLabel id="refered-label">
+                {preDischargeForm.discharge_reason == "EXP"
+                  ? "Cause of death"
+                  : "Discharge notes"}
+              </InputLabel>
```

Comment:
Make the field `mandatory` when discharge reason is `EXP`.
```suggestion
              <InputLabel id="refered-label" required={preDischargeForm.discharge_reason == "EXP"} >
                {preDischargeForm.discharge_reason == "EXP"
                  ? "Cause of death"
                  : "Discharge notes"}
              </InputLabel>
```

Add validations for the same

Reply:
Done.